jumpscale/packages/vdc_dashboard/bottle/api/deployments.py

```diff
@@ -420,13 +420,13 @@ def accept():
 @app.route("/api/update", method="GET")
 @package_authorized("vdc_dashboard")
 def update():
-    branch_param = request.params.get("branch")
-    if branch_param:
-        ref = branch_param
+    ref_param = request.params.get("ref")
```

Comment:
I believe in keeping the branch option and adding the ref beside, it is helpful in the dev process. what do you think?

Reply:
I think the branch also will work in this case, I will approve :+1:

Reply:
ref could be a branch name or tag name, these two use cases work equally
/update?ref=development  // branch name
/update?ref=v11.5-b9  // tag name